test/inductor/test_select_algorithm.py

```diff
@@ -344,9 +344,48 @@ def test_TritonTemplateCaller_str(self):
         caller_str = str(caller)
         self.assertEqual(caller_str, f"TritonTemplateCaller({module_path}, extra)")
 
+    @patch.object(select_algorithm, "VERIFY", dict(atol=5e-2, rtol=5e-2))
+    @patches
+    def test_mm_fp16_acc(self):
+        cfg = torch.backends.cuda.matmul
+        restore = cfg.allow_fp16_reduced_precision_reduction
+        try:
```

Comment:
patch.object doesn't work because of `__{get, set}_attr__` override defined on matmul object. Instead, used try except to restore.